mongo_proxy/durable_cursor.py

```diff
@@ -227,9 +235,8 @@ def try_reconnect(self, get_next=True):
         raise MongoReconnectFailure()
 
     def count(self, with_limit_and_skip=False):
-        while True:
-            try:
-                return self.cursor.count(
-                    with_limit_and_skip=with_limit_and_skip)
-            except AutoReconnect:
-                self.try_reconnect(get_next=False)
+        return self._with_retry(
```

Comment:
I don't know why we'd want different behavior between a count and any other generic operation.